src/core/lib/layer.js

```diff
@@ -749,11 +725,11 @@ ${flags.viewportChanged ? 'viewport' : ''}\
     }
 
     // TODO - set needsRedraw on the model(s)?
-    this.state.needsRedraw = true;
+    this.setNeedsRedraw();
     log.deprecated('layer.setUniforms', 'model.setUniforms');
   }
 }
 
 Layer.layerName = 'Layer';
-Layer.propTypes = defaultProps;
-Layer.defaultProps = defaultProps;
+Layer.propTypes = LayerProps.defaultProps;
```

Comment:
`propTypes`?

Reply:
> propTypes?
Good catch, missed removing this when splitting the diff...